Makefile

```diff
@@ -184,9 +171,9 @@ $(OUT)/treasuryDAO_storage.tz : slash_division_value = 0n
 $(OUT)/treasuryDAO_storage.tz : min_xtz_amount = 0mutez
 $(OUT)/treasuryDAO_storage.tz : max_xtz_amount = 100mutez
 $(OUT)/treasuryDAO_storage.tz : now_val = `date +"%s"`
-$(OUT)/treasuryDAO_storage.tz : metadata_map = (Big_map.empty : metadata_map)
+$(OUT)/treasuryDAO_storage.tz : metadata_map = Big_map.empty
 $(OUT)/treasuryDAO_storage.tz : fixed_proposal_fee_in_token = 0n
-$(OUT)/treasuryDAO_storage.tz : ledger = ([] : ledger_list)
+$(OUT)/treasuryDAO_storage.tz : ledger = []
```

Comment:
Even if you revert the changes below (as I suggested in the other comment) to remove the use of `require_defined` that are not necessary, I think we should keep this (and move only the type annotation in the call below).

It seems to me it makes things cleaner overall.